exonum-java-proofs/src/test/java/com/exonum/binding/hash/HashingTest.java

```diff
@@ -1,4 +1,19 @@
-/* 
+/*
```

Comment:
Probably, ours is first.

Reply:
In other guava classes we have Guava license first, ours second. Do you think we should change it there as well?